gcs/gcs-generic.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+ name: gcs-generic
+spec:
+  workspaces:
+  - name: credentials
+    description: A secret with a service account key to use as GOOGLE_APPLICATION_CREDENTIALS.
+  params:
+  - name: command
+    description: The command line that you would like to enter.
+    type: string 
+  - name: subcommand
+    description: The subcommand line that you would like to enter.
+    type: array 
+  - name: serviceAccountPath
+    description: |
+      The path inside the credentials workspace to the GOOGLE_APPLICATION_CREDENTIALS key file.
+    type: string
+    default: service_account.json
+  steps:
+  - name: run-gsutil
+    image: google/cloud-sdk
```

Comment:
Let's add another parameter that defaults to this to allow users to pass in their own image.

This will help if people want to pin to certain gsutil versions.

Reply:
that's a good idea. We should do it for the other gcs tasks too 😃  what do you think @sbwsg?

Reply:
added a parameter for image and set default as "google/cloud-sdk"